mythril/laser/ethereum/transaction/symbolic.py

```diff
@@ -17,14 +18,33 @@
 
 log = logging.getLogger(__name__)
 
-CREATOR_ADDRESS = 0xAFFEAFFEAFFEAFFEAFFEAFFEAFFEAFFEAFFEAFFE
-ATTACKER_ADDRESS = 0xDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEF
+ACTOR_ADDRESSES = {
```

Comment:
How about using a dataclass over a Dict(), It might take some effort now, but our future would be simple due to autocomplete.

Reply:
The reason I initially went with a dict is so that we could be flexible with adding other addresses, if for example we wanted to add a `SOMEGUY2`. I can use a class with a property for the `creator` and `attacker` address and have that look it up in its dict though.

Reply:
Yeah, this should be cool.